package.json

```diff
@@ -45,7 +45,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "3.6.0",
+  "version": "3.6.1",
```

Comment:
Let's drop the version change and release it together in 3.6.0

Reply:
:+1: